--- ../../portal/src/game/playerbot/PlayerbotMgr.cpp	2014-07-06 12:14:56.000000000 -0400
+++ src/game/playerbot/PlayerbotMgr.cpp	2014-07-19 16:57:35.000000000 -0400
@@ -288,12 +288,14 @@
         // such as inventory that can be equipped
     case CMSG_INSPECT:
         {
+            /* >>> Moved to textemote /curious - was too annoying
             WorldPacket p(packet);
             p.rpos(0); // reset reader
             ObjectGuid guid;
             p >> guid;
             Player* const bot = GetPlayerBot(guid);
             if (bot) bot->GetPlayerbotAI()->SendNotEquipList(*bot);
+            */
             return;
         }
 
@@ -379,6 +381,15 @@
                     return;
                 }
 
+
+            // emote to show inventory that can be equipped
+            case TEXTEMOTE_CURIOUS:
+                {
+                    Player* const bot = GetPlayerBot(m_master->GetSelectionGuid());
+                    if (bot) 
+                        bot->GetPlayerbotAI()->SendNotEquipList(*bot);
+                }
+
                 // emote to attack selected target
             case TEXTEMOTE_POINT:
                 {
